kratos/solving_strategies/strategies/line_search_strategy.h

```diff
@@ -562,7 +562,7 @@ class LineSearchStrategy
         const bool CalculateReactions
         )
     {
-        rDefaultSettings["max_iterations"].SetInt(MaxIterations);
+        rDefaultSettings["max_iteration"].SetInt(MaxIterations);
```

Comment:
I think we had a discussion related with this. Which name is used in the python solvers?

Reply:
withoit the s.

as it is now it fails